osu.Game.Tests/Visual/Multiplayer/TestSceneMatchHeader.cs

```diff
@@ -7,46 +7,53 @@
 using osu.Game.Rulesets.Osu;
 using osu.Game.Rulesets.Osu.Mods;
 using osu.Game.Screens.OnlinePlay.Match.Components;
+using osu.Game.Tests.Visual.OnlinePlay;
 using osu.Game.Users;
 
 namespace osu.Game.Tests.Visual.Multiplayer
 {
-    public class TestSceneMatchHeader : RoomTestScene
+    public class TestSceneMatchHeader : OnlinePlayTestScene
     {
-        public TestSceneMatchHeader()
+        [SetUp]
+        public new void Setup() => Schedule(() =>
         {
+            SelectedRoom.Value = new Room();
+
             Child = new Header();
-        }
+        });
 
-        [SetUp]
-        public new void Setup() => Schedule(() =>
+        [Test]
+        public void TestBasicRoom()
         {
-            Room.Playlist.Add(new PlaylistItem
+            AddStep("set basic room", () =>
             {
-                Beatmap =
+                SelectedRoom.Value.Playlist.Add(new PlaylistItem
```

Comment:
Unrelated, but this test is quite the fake-out - the component being tested does actually literally nothing with the playlist item or its mods, so I wrongly thought something broke here for a second >.>

Reply:
You're completely right. This is one of the tests I wrote early on, which required a bit more complex testing structure.